interfaces/Portalicious/src/app/components/dynamic-form/dynamic-form.component.html

```diff
@@ -0,0 +1,50 @@
+<form
+  class="block w-full"
+  [formGroup]="formGroup()"
+  (ngSubmit)="onFormSubmit()"
+>
+  <div class="flex flex-col gap-2">
+    @for (field of fields(); track field.controlName) {
+      <!-- eslint-disable-next-line @angular-eslint/template/label-has-associated-control -->
+      <label>
+        <ng-container>{{ field.label }}</ng-container>
+        @switch (field.type) {
+          @case ('password') {
+            <p-password
+              [formControlName]="field.controlName"
+              [autocomplete]="field.autocomplete"
+              [toggleMask]="true"
+              [feedback]="false"
+              styleClass="mt-2 w-full [&_input]:w-full"
+            />
+          }
+          @default {
+            <!-- eslint-disable @angular-eslint/template/no-autofocus -->
+            <input
```

Comment:
Should we use the standard input or is it better to use PrimeNG components as much as possible?

Reply:
I'm not sure I understand the question 😬  What would "using the standard input" mean?

Anyways, without understanding the question, 

> is it better to use PrimeNG components as much as possible

my answer to this would be yes.